docs/spec.md

```diff
@@ -6268,6 +6294,19 @@ If `x` is a value or placeholder, this function is a shortcut for
 `member_name(type(x))`.  If `x` is not a type that has an appropriate member, or
 a value or a placeholder of such a type, returns `None`.
 
+* `is_convertible_or_quantizable(x: Type, y: Type) -> bool` checks for the
+equality of `x` and `y`, ignoring the bitwidth, when they are of type
```

Comment:
I think we should have something more than equality. eg. compatibility for quantization/dequantization

Reply:
Hi @doyeonkim0 
I agree that this is a restrictive version of equality and also the name `is_convertible_quantizable` might cause confusion as it as not considering all possible scenarios.  How about we use `is_promotable` and add justification in the function definition about the rationale of using it.